xdp/src/device.rs

```diff
@@ -86,7 +86,8 @@ impl NetworkDevice {
             );
         }
 
-        let result = unsafe { ioctl(fd.as_raw_fd(), SIOCGIFHWADDR, &mut req) };
+        let result =
+            unsafe { libc::syscall(libc::SYS_ioctl, fd.as_raw_fd(), SIOCGIFHWADDR, &mut req) };
```

Comment:
@alessandrod LKM if you agree with that. That's exactly how we fixed musl build in Aya as well.

Reply:
use libc::{syscall, SYS_ioctl} instead of full paths? otherwise looks fine